### PR TITLE
Added the Profile update page

### PR DESCRIPTION
**Pull request by Badr-KH**
Users are redirected to the profile update page after they sign up !

### CODE REVIEW

File: client/src/pages/SignupForm.js [hunk 1]
```diff
@@ -55,15 +56,40 @@ export default function SignupForm() {
...
+      .then(res => {
+        if (res.error) {
+          const formErrors = { ...error };
+          formErrors.message = res.error;
+          setErrors(formErrors);
+          return;
+        }
+        if (res.accessToken) {
+          localStorage.setItem("authToken", res.accessToken);
+          window.location.href = "/Update?welcome=true";
```
Comment (aeciorc) [inline client/src/pages/SignupForm.js]: use `props.history` instead

File: client/src/App.js [hunk 2]
```diff
@@ -1,52 +1,57 @@
...
 import "./App.css";
+import ProfileUpdate from "./pages/ProfileUpdate";
 
 
 function App() {
-  // when true, user logged in and will not be redirected to dashboard
-  let userLoggedIn = false;
-
-  let loggedIn = (
+  const [auth, setAuth] = useState({
```
Comment (aeciorc) [inline client/src/App.js]: I think this whole logic could be a custom hook that is declared elsewhere. 
Read this: https://reactjs.org/docs/hooks-custom.html

**Review by aeciorc — APPROVED** [review]
Great job. One small piece of feedback: let's avoid having too many features on a single PR; if they're unrelated, they should be in different PRs. It also seems like you add too many changes per a single commit, try to make more commits (one per milestone). That'll make it easier to revert one of them in the future without affecting the rest.

**Review by Badr-KH — COMMENTED** [review]
Will do, thanks for your feedback Aecio.

**Pull request merged**